docs-site/src/components/Search/EmptySearch.tsx

```
@@ -0,0 +1,137 @@
+            {recentSearches.map((query) => (
+              <li key={query}>
+                <div
+                  className={styles.recentSearchItem}
+                  onClick={() => onRecentSearchClick(query)}
+                  onKeyDown={(e) => {
+                    if (e.key === "Enter" || e.key === " ") {
+                      e.preventDefault();
+                      onRecentSearchClick(query);
+                    }
+                  }}
+                  role="button"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)